paderbox/array/interval/core.py

```diff
@@ -166,32 +163,38 @@ class ArrayInterval:
     from_str = staticmethod(ArrayInterval_from_str)
     inverse_mode = False
 
-    def __init__(self, array, inverse_mode=False):
+    def __init__(self, array, inverse_mode: bool = False):
         """
 
+
+        The `ArrayInterval` is in many cases equivalent to a 1-dimensional
+        boolean numpy array that stores activity information in an efficient
+        way.
+
         Args:
             array: 
             inverse_mode:
                 Flag to indicate what the intervals represent:
-                 - False: Intervals represent True
-                 - True: Intervals represent False
+                 - False: Intervals represent activity (i.e., `True`)
+                 - True: Intervals represent silcence (i.e., `False`)
```

Comment:
The `inverse_mode` flag is an internal detail and is necessary to implement `np.ones` with an unknown length.
It has nothing to do with `activity` or `silence`.